compiler/src/dmd/dmodule.d

```diff
@@ -1173,15 +1172,15 @@ extern (C++) final class Module : Package
     }
 
     // Back end
-    Symbol* cov; // private uint[] __coverage;
+    void* cov; // private uint[] __coverage;
     uint[] covb; // bit array of valid code line numbers
-    Symbol* sictor; // module order independent constructor
-    Symbol* sctor; // module constructor
-    Symbol* sdtor; // module destructor
-    Symbol* ssharedctor; // module shared constructor
-    Symbol* sshareddtor; // module shared destructor
-    Symbol* stest; // module unit test
-    Symbol* sfilename; // symbol for filename
+    void* sictor; // module order independent constructor
+    void* sctor; // module constructor
+    void* sdtor; // module destructor
+    void* ssharedctor; // module shared constructor
+    void* sshareddtor; // module shared destructor
+    void* stest; // module unit test
```

Comment:
IIUC, all these are now unused?

Reply:
no, not yet. https://github.com/dlang/dmd/blob/master/compiler/src/dmd/glue.d#L1339 also some of them are used by GDC ;) and by the `cxxfrontend.cc` test

Reply:
You can turn it into a bit flag then!

Don't worry about gdc here, these can be moved to a private struct.

https://github.com/gcc-mirror/gcc/blob/3a7fcf4f54ecffdbad03787d4f734c1fb2291ef5/gcc/d/modules.cc#L76-L79

Reply:
```
struct modbitflags {
  bool sictor, sctor, sdtor, ssharedctor, sshareddtor;
}
mixin(generateBitFields!(modbitflags, ubyte));
```